tests/foreman/destructive/test_installer.py

```diff
@@ -73,8 +73,9 @@ def test_installer_sat_pub_directory_accessibility(target_sat):
     command_output = target_sat.execute('satellite-installer', timeout='20m')
     assert 'Success!' in command_output.stdout
     for command in [http_curl_command, https_curl_command]:
-        accessibility_check = target_sat.execute(command)
-        assert 'HTTP/1.1 200 OK' or 'HTTP/2 200 ' not in accessibility_check.stdout.split('\r\n')
+        accessibility_check = target_sat.execute(command).stdout.split('\r\n')
+        assert 'HTTP/1.1 200 OK' not in accessibility_check
+        assert 'HTTP/2 200 ' not in accessibility_check
```

Comment:
Same recommendation here
```suggestion
        accessibility_check = target_sat.execute(command).stdout
        assert 'HTTP/1.1 200 OK' not in accessibility_check
        assert 'HTTP/2 200 ' not in accessibility_check
```